### ISSUE TITLE
Documentation in FAQ is for unreleased 1.14 instead of 1.13

### ISSUE THREAD

**Issue by benoit74** [openzim/mwoffliner#2032]
Documentation at https://github.com/openzim/mwoffliner/wiki/Frequently-Asked-Questions seems to speak about CLI parameters which do not yet exists in the Zimfarm, making it particularly difficult to use for our editors.

Could we fix this to have a documentation aligned with what is used in production (or release 1.14? 😉)

**Comment by Jaifroid** [comment]
Agree, but surely the solution is to release 1.14 asap? It seems quite urgent to me. Our Wikipedia ZIMs are in a pretty ramshackle state right now: titles are missing from all articles (#1872, despite being fixed 8 months ago in #1876), there is HTML in `dirEntry,title` fields (#1797: although this is a Phabricator bug, it would be simple to work around in the scraper), Wiktionary translations are still hidden (#1033 - again, the workaround is known)... NB These are just observations, I'm not saying anyone in particular is responsible as I know it's been hard to get sustained development for mwOffliner. But it seems to me that mwOffliner is our "bread and butter", so could do with some more love!

**Comment by kelson42** [comment]
Plan is to solve this issue by releasing 1.14.0

**Comment by benoit74** [comment]
> Plan is to solve this issue by releasing 1.14.0

I love this!

**Comment by audiodude** [comment]
> I'm not saying anyone in particular is responsible

I think that's precisely the problem! 😅
